### PR TITLE
Pin dependencies & Rename

### PR DESCRIPTION
**Pull request by gluck**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "baseUrl": ".",
+    "outDir": "./dist",
```
Comment (bashmish) [inline tsconfig.json:12]: can this break typings (as you explain in Discord)? or is this starter-kit not using them?

File: tsconfig.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
     "paths": {
       "~/*": ["*"]
```
Comment (bashmish) [inline tsconfig.json:14]: shouldn't this be?

```suggestion
      "~/*": ["*"],
      "@divriots/starter-origami/*": ["*"]
```

looks like I'm the only one so far who configured this in my starter-kit, but not getting why so

Comment (gluck) [reply tsconfig.json:14]: :+1:

File: tsconfig.json (hunk 1 above)
Comment (gluck) [reply tsconfig.json:12]: This one hasn't been released to npm, so no impact yet, and I have to fix it anyway

Comment (gluck) [reply tsconfig.json:12]: Checked typings aren't impacted, they ignore tsconfig (probably wrongly so)

File: tsconfig.json (hunk 2 above)
Comment (gluck) [reply tsconfig.json]: done

File: tsconfig.json (hunk 1 above)
Comment (bashmish) [reply tsconfig.json:12]: 👍

**Review by bashmish — APPROVED** [review]

**Pull request merged**